config/routes.rb

```diff
@@ -5,59 +5,8 @@
 
   devise_for :users
   resources :metrics
+  resources :features
```

Comment:
I did try to do this but I was getting a lot of problems. Rails kept complaining about routes like features/index not being available when I was creating a new feature, even though I was redirecting the user to '/'. It definitely needs more looking into on my part, thank you.